Library/Homebrew/global.rb

```diff
@@ -60,6 +60,8 @@
 HOMEBREW_DEFAULT_PREFIX = "/usr/local"
 LINUXBREW_DEFAULT_PREFIX = "/home/linuxbrew/.linuxbrew"
 
+HOMEBREW_REQUIRED_RUBY_VERSION = ENV["HOMEBREW_REQUIRED_RUBY_VERSION"]
```

Comment:
Maybe move this higher to align with the other constants set from `ENV`? This could also be used by `check_ruby_version` in `Library/Homebrew/extend/os/mac/diagnostic.rb`